pennylane/utils.py

```diff
@@ -433,3 +434,77 @@ def expand_vector(vector, original_wires, expanded_wires):
     expanded_tensor = np.moveaxis(expanded_tensor, original_indices, wire_indices)
 
     return expanded_tensor.reshape(2 ** M)
+
+
+class SneakPeak:
```

Comment:
The spelling is `SneakPeek`, but for some reason that doesn't look as nice :frowning_face:

Reply:
Whoops